Ginger/GingerCoreNET/CentralExecutionLogger/CentralExecutionLoggerHelper.cs

```diff
@@ -0,0 +1,150 @@
+﻿using AccountReport.Contracts;
+using amdocs.ginger.GingerCoreNET;
+using Amdocs.Ginger.Common;
+using Amdocs.Ginger.CoreNET.LiteDBFolder;
+using AutoMapper;
+using Ginger.Reports;
+using RestSharp;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Amdocs.Ginger.CoreNET.CentralExecutionLogger
+{
+    public class CentralExecutionLoggerHelper
+    {
+     
+
+        private string EndPointUrl { get; set; }
+
+        RestClient restClient;
+        public CentralExecutionLoggerHelper(string apiUrl)
+        {
+            EndPointUrl = apiUrl;          
+            restClient = new RestClient(apiUrl);
+            restClient.RemoteCertificateValidationCallback = (sender, certificate, chain, sslPolicyErrors) => true;
+        }
+
+        public AccountReportRunSet MapDataToAccountReportObject(LiteDbRunSet runSet)
+        {
+            AccountReportRunSet accountReportRunSet = new AccountReportRunSet();
+
+            var config = new MapperConfiguration(cfg =>
+            {
+                cfg.CreateMap<LiteDbReportBase, AccountReportBase>().ForMember(dest => dest.ElapsedEndTimeStamp, src => src.MapFrom(x => x.Elapsed)).IncludeAllDerived();
+                cfg.CreateMap<LiteDbRunSet, AccountReportRunSet>().ForMember(dest => dest.ExecutedByUser, src => src.MapFrom(x => x.ExecutedbyUser));
+                cfg.CreateMap<LiteDbRunner, AccountReportRunner>();
+                
+                cfg.CreateMap<LiteDbBusinessFlow, AccountReportBusinessFlow>();
+                cfg.CreateMap<LiteDbActivityGroup, AccountReportActivityGroup>();
+                cfg.CreateMap<LiteDbActivity, AccountReportActivity>();
+                cfg.CreateMap<LiteDbAction, AccountReportAction>();
+                //cfg.CreateMap<KeyValuePair<string, int>, AccountReport.Contracts.Helpers.DictObject>().ConvertUsing(x =>
+                cfg.CreateMap<KeyValuePair<string, int>, AccountReport.Contracts.Helpers.DictObject>()
+                .ForMember(dest => dest.Key, src => src.MapFrom(x => x.Key))
+                .ForMember(dest => dest.Value, src => src.MapFrom(x => x.Value));
+
+            });
+
+            IMapper iMapper = config.CreateMapper();
+
+            var destination = iMapper.Map<LiteDbRunSet, AccountReportRunSet>(runSet);
+            
+            //destination.ElapsedEndTimeStamp = runSet.Elapsed;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5632404)